bin/inference/pycbc_inference_extract_samples

```diff
@@ -158,6 +212,11 @@ for fp in fps:
 # store what parameters were renamed
 out.attrs['remapped_params'] = list(labels.items())
 
+# write combined evidence and dlog evidence if combining via sampling
+if opts.combine_via_sampling:
+    out.attrs['log_evidence'] = logsumexp(logz_list)
+    out.attrs['dlog_evidence'] = numpy.sqrt(sum([i**2 for i in dlogz_list]))
+
```

Comment:
There's an issue with this: you only want to add evidence in this way if the priors are mutually exclusive. If someone was combining two files that have the same prior, the evidence would be averaged. (And if the priors are partially overlapping, they shouldn't be combining at all.) In principle, combine via sampling could be done in either case. Maybe add another option called `mutually-exclusive-priors`, and if that's set, add the evidences; otherwise average them. Make sure to put in the help message that if `mutually-exclusive-priors` is not set it's assumed that the priors are the same. Also, as stated above, only do all of this if  log evidence exists in the file.